bitnami/thanos/Chart.yaml

```diff
@@ -15,4 +15,4 @@ maintainers:
 name: thanos
 sources:
   - https://github.com/bitnami/bitnami-docker-thanos
-version: 0.5.3
+version: 0.5.4
```

Comment:
I would say it should be a minor, it is a new feature for the chart.

Reply:
Thank you @dani8art I pushed an updated tree